src/bitstream.rs

```diff
@@ -147,13 +158,13 @@ impl<'a> Bitstream<'a> {
         Ok(hi << 8 | lo)
     }
 
-    pub fn align(&mut self) -> bool {
+    pub fn align(&mut self) -> Result<(), DecodeFailed> {
```

Comment:
Maybe we should rename the method to make it clear that this operation will "trash" some data.

Reply:
Well, it doesn't look like trash, for me. And it was easier to debug, because I saw similar things in different code)

Reply:
I mean trash in the sense that it reads 16 bits and drops them ("trashing" them, being unable to access them).

Reading `align`, it wouldn't be my first instinct that it does this. But maybe the name is fine.